dipy/viz/colormap.py

```diff
@@ -255,3 +261,52 @@ def line_colors(streamlines, cmap='rgb_standard'):
                     for streamline in streamlines]
 
     return np.vstack(col_list)
+
+
+lowercase_cm_name = {'blues': 'Blues', 'accent': 'Accent'}
+
+
+def create_colormap(v, name='jet', auto=True):
```

Comment:
From top to bottom. Inferno, jet and viridis on a response function. Clearly jet shows the minima better. But the other have a smoother transition.

![inferno](https://cloud.githubusercontent.com/assets/134276/19987833/75213fd4-a1f3-11e6-8455-487ad0488ca1.png)
![jet](https://cloud.githubusercontent.com/assets/134276/19987834/781f18dc-a1f3-11e6-8291-f28483852991.png)
![viridis](https://cloud.githubusercontent.com/assets/134276/19987836/7b36c826-a1f3-11e6-9045-7f9352b30b2b.png)